src/shared/dlt_common.c

```diff
@@ -1793,6 +1795,20 @@ void dlt_log_set_filename(const char *filename)
     logging_filename[NAME_MAX] = 0;
 }
 
+void dlt_log_set_filemode(int filemode)
+{
+    if (filemode == DLT_OVERWRITE_LOG_FILE) {
+        strncpy(logging_filemode, log_to_file_mode_list[1], 1);
```

Comment:
Please use enum instead of magic number here.
I can see you define the enum in header file so I guess it should be used.

Reply:
Sure, thanks for your notice